chainList.json

```diff
@@ -408,4 +425,4 @@
       "chain": "sepolia-dev-0"
```

Comment:
Can a platform engineer reviewer take a look at why this was against `sepolia-dev-0` instead of `sepolia`?

Reply:
I think it should just be `sepolia` but I don't have a ton of context

Reply:
very weird

Reply:
you're commenting on something other than our chain definition

Reply:
Yep, you're right, sorry I'm on mobile. Overall the PR looks good, it looks like there's a test failure because of a missing newline

Reply:
We're also addressing the 1s blocktime check

Reply:
After I can get an approver and get this pr merged, you should be able to rebase and get the blocktime validation checks to pass: https://github.com/ethereum-optimism/superchain-registry/pull/631